src/makeRobotAccountant.js

```diff
@@ -6,7 +6,25 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  const getSum = (a) => {
+    const add = (b) => {
+      if (count === 3) {
+        count--;
+
+        return 'Bzzz... Error!';
+      }
+
+      count++;
+
+      return a + b;
```

Comment:
The add function should return the sum of a and b regardless of the count value. It's better to put this return statement outside the condition.